core/src/dird/ua_cmds.cc

```diff
@@ -587,16 +587,16 @@ bool Do_a_command(UaContext* ua)
   len = strlen(ua->argk[0]);
   for (i = 0; i < comsize; i++) { /* search for command */
     if (bstrncasecmp(ua->argk[0], commands[i].key, len)) {
+      const char* command = commands[i].key;
       // Check if command permitted, but "quit" and "whoami" is always OK
-      if (!bstrcmp(ua->argk[0], NT_("quit"))
-          && !bstrcmp(ua->argk[0], NT_("whoami"))
-          && !ua->AclAccessOk(Command_ACL, ua->argk[0], true)) {
+      if (!bstrcmp(command, NT_("quit")) && !bstrcmp(command, NT_("whoami"))
+          && !ua->AclAccessOk(Command_ACL, command, true)) {
```

Comment:
What about the "exit" command ? We should probably also allow that one.

Im not sure why the quit/exit commands are even sent to the director in the first place.  They should be handled locally by the console but i digress.

Reply:
After an internal discussion, we decided to leave it as it is for the moment, at least don't change it with this PR:

Reply:
@joergsteffens but i think we wanted to add exit here as well.

Reply:
okay